main.rb

```diff
@@ -0,0 +1,42 @@
+# ---------------- Bubble Sort Method ------------------------ /
+
+def bubble_sort(arr)
+  swapped = false
+  runs = 1
+  until swapped
+    swapped = true
+    (0...arr.length - runs).each do |i|
+      next unless arr[i] > arr[i + 1]
+
+      swapped = false
+      temp = arr[i]
+      arr[i] = arr[i + 1]
+      arr[i + 1] = temp
```

Comment:
- [ ] **OPTIONAL** You can also use ruby's easy swap in one line, it's like this:
`x,y = y,x`